userguide/storage.rst

```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop
+down. Choose a disk from the list of :guilabel:`Available disks` and
+click :guilabel:`+` to add that disk to the volume. Select *spare* in
+the :guilabel:`Volume layout` drop down. Click :guilabel:`Extend Volume`
+to finalize the process of adding a hot spare to the selected volume.
```

Comment:
```suggestion
to add the hot spare to the selected volume.
```

Reply:
Oops, was missing a word, meant to say "to add the hot spare."  I think context makes it unnecessary to repeat "to the selected volume".